src/utils/hints/stateChannelsHints.js

```diff
@@ -0,0 +1,9 @@
+export const stateChannelsHints = {
+  stateChannelId: 'Unique identifier of the State Channel.',
+  status: 'When status is "Open" transactions can be executed inside the State Channel, outside the blockchain network.',
+  participants: 'Initiator is the account that created the State Channel via ChannelCreateTx.',
+  onChainUpdates: 'Amount of on-chain transactions within the State Channel.',
+  locked: 'Currently deposited amount to the State Channel by its participants.',
+  lastUpdated: 'Estimated date, time, and block height of the last interaction with the State Channel.',
```

Comment:
Same for other PRs, could you please adjust it to 
"Block height and estimated date and time of the last interaction with the State Channel."

Reply:
I created follow-up on this as it is already used in merged parts https://github.com/aeternity/aescan/issues/307
But I also fixed it here.